common/middleware/ensure-authenticated.js

```diff
@@ -23,7 +23,7 @@ module.exports = function ensureAuthenticated({
     req.session.originalRequestUrl = req.originalUrl
 
     if (req.method === 'POST') {
-      const contentType = req.header('content-type')
```

Comment:
Maybe add a specific test for the scenario where the fallback is used?

Reply:
Resolved by returning undefinded from base stub